### ISSUE TITLE
Proposal for Prometheus Metric to Monitor Dropped Packets

### ISSUE THREAD

**Issue by misaki-kawakami** [dmachard/go-dnscollector#641]
**Is your feature request related to a problem? Please describe.**
At present, we lack an easy way to monitor number of dropped packets by a logger when the buffer is full. A a new Prometheus metric would improve our monitoring capabilities, allowing us to trigger alerts if a significant number of packets are being dropped.

**Describe the solution you'd like**
A new Prometheus metric that can indicate the number of dropped packets per logger/collector. This metric provides insight into the number of dropped packets, eliminating the need to search through logs.

**Describe alternatives you've considered**
Checking dns-collector logs every now and then but with large number of instances, this method is becoming impractical.

**Comment by dmachard** [comment]
Good idea, I think adding global  metrics to DNScollector would be the right way to expose this type of information.

**Comment by dmachard** [comment]
The following new metric will be available in next release.

```
# HELP dnscollector_exporter_worker_discarded_traffic_total Discarded traffic associated to each worker
# TYPE dnscollector_exporter_worker_discarded_traffic_total counter
dnscollector_exporter_worker_discarded_traffic_total{worker="console"} 0
dnscollector_exporter_worker_discarded_traffic_total{worker="tap"} 0
````

**Comment by dmachard** [comment]
![image](https://github.com/dmachard/go-dnscollector/assets/5562930/6143532d-b434-424d-943b-6f9499801e7e)
